test/integration/spec/localparticipant/regressions.js

```diff
@@ -33,7 +33,7 @@ const {
   waitForTracks
 } = require('../../../lib/util');
 
-describe('LocalParticipant: regresions', function() {
```

Comment:
are all tests in this suite unstable? if not please mark only specific tests that are unstable.